dockers/docker-database/docker-database-init.sh

```diff
@@ -125,6 +129,12 @@ do
     else
         echo -n > /var/lib/$inst/dump.rdb
     fi
+
+    chown -R redis:redis /var/lib/$inst
```

Comment:
Add comments to this file, help other understand.
Also, this seems not related with enable multi-DB, can this move to another PR?

Reply:
Here are the necessary permissions for enable_multidb, see /etc/supervisor/conf.d/supervisord.conf.j2

Reply:
Hi @Pan-XT , can you add your comments to this file? that will help other understand the propose of this code.

Reply:
ok，I add commnets “the Redis process is operating under the 'redis' user in supervisord and make redis user own /var/lib/$inst inside db container.”

Reply:
#Closed